styleguide/_themes/derek/scss/patterns/videoEmbed.scss

```diff
@@ -0,0 +1,14 @@
+.videoEmbed-title {
+  text-align: center;
+}
+
+.videoEmbed-row {
+  @include make-row;
+}
+
+// when the make column is used in the sass then it overrides any center-block declaration with a float:left ??
+.videoEmbed-column {
+  @include make-md-column(6)
+  @include center-block;
+  padding: $half-padding-full;
+}
```

Comment:
Do you need a semicolon at the end of `@include make-md-column(6)`?

Not sure if that affects anything, but worth a shot.